src/content/en/updates/2017/06/supercharged-codesplit.md

```diff
@@ -0,0 +1,279 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: We implemented code splitting and route based chunking just like WebPack.
+
+{# wf_updated_on: 2017-06-22 #}
+{# wf_published_on: 2017-06-22 #}
+{# wf_tags: supercharged #}
```

Comment:
I agree and would add:
* If there are other supercharged articles, submit a PR to add the tag to them as well. This will result in a URL listing all of the supercharged article.
* Add some tags that are about your content rather than your branding. Reuse existing tags when possible. If nothing fits, ping me for guidance.